R/user_helpers.R

```diff
@@ -1,5 +1,7 @@
 #' Open an ADaM Template Script
 #'
+#' Uses list_all_templates() to discover which templates are available
```

Comment:
Please move this to the `@details` section. Also please put the function name into backticks. Plus it should not be Use~s~ but just Use.

Reply:
done